dashboard/src/containers/Helpers.js

```diff
@@ -98,6 +81,27 @@ export function getNodeLabel(properties: Object, regex: string): string {
   return label;
 }
 
+function getNodeLabel(properties: Object, regex: string): string {
+  var label = "";
+
+  let keys = Object.keys(properties);
+  if (keys.length === 1) {
+    label = aggregationPrefix(properties)[0];
+    if (label !== "") {
+      return label;
+    }
+  }
+
+  let nameKey = getNameKey(properties, regex);
+  if (nameKey === "") {
+    return "";
+  }
```

Comment:
maybe we can simply do:

    return properties[nameKey] || '';

or if returning an empty string is not that important,

    return properties[nameKey];